app/components/settings/billing-page/renewal-section.hbs

```diff
@@ -0,0 +1,8 @@
+<Settings::FormSection @title="Renewal" @description="Details of your upcoming payment">
+  <Settings::BillingPage::StatusPill @color="gray" class="mb-3">
```

Comment:
Just fyi, this `mb-3` isn't actually taking effect since there's `mb-4` within the child component and that has higher precedence. Once we do move `mb-4` out into the template, this will start taking effect and things are going to look inconsistent. 

Let's use `mb-3` everywhere, that's what the design PR used if I'm not wrong, and just tested on the deployment preview and that looks neater